contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,77 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+import logging
+from google import search
+
+def get_google_result(original_content):
+    search_keywords = original_content.split(' ', 1)[1]
+    if search_keywords == '--help':
```

Comment:
I'm not sure why you're using `--help` here, wouldn't `@google help` be a bit easier to use? I don't think anyone will search for just `help`.